client/src/components/Header.js

```diff
@@ -4,6 +4,7 @@ export default function Header() {
   return (
     <header className="header">
       <NavLink to="/">Home</NavLink>
+      <NavLink to="/2">Home2</NavLink>
```

Comment:
thank you for the reviews john & namir! this pull request was probably not a good idea to post, since i just was experimenting with stuff! Thank you though for your comments, they really help!

Reply:
@mikemangas you can always post PRs for experimenting but its important you describe that in the description, so reviewers know this is not going to be merged but you still want an opinion on something :)

Reply:
thanks for the info!